docs/source/index.mdx

```diff
@@ -76,18 +75,20 @@ The difference is that now, each subgraph defines the types and fields that it i
 
 The result is the best of both worlds: an implementation that keeps all the code for a given feature in a single subgraph and separated from unrelated concerns, and a product-centric schema with rich types that reflects the natural way an application developer would want to consume the graph.
 
-## Apollo Server libraries
+## Federated schemas
 
-Apollo Server supports Apollo Federation via two open-source extension libraries:
+A federated graph uses multiple "types" of GraphQL schemas:
 
-* `@apollo/federation` provides primitives that subgraphs use to make their individual GraphQL schemas composable.
-* `@apollo/gateway` enables you to set up an instance of Apollo Server as a **gateway** that distributes incoming GraphQL operations across one or more  subgraphs.
+* **Subgraph schemas.** Each of your **subgraphs** has its own schema that represents the types and fields of your **supergraph** that it's responsible for resolving.
```

Comment:
I think this needs just a just a bit more clarity. Might even add one more statement that says collectively, the subgraphs must be capable of resolving the entire supergraph.

Edit: the addition I'm proposing _is_ mentioned immediately below, so maybe disregard.

```suggestion
* **Subgraph schemas.** Each of your **subgraphs** has its own schema that represents some of the types and fields of your **supergraph** that it's responsible for resolving.
```